### DISCUSSION TITLE
HTTP ERROR 500

### DISCUSSION THREAD

**Discussion by alex7070m** [rectorphp/rector#8762] [Q&A]
Hello!
Rector processed the script files for php 8.2, but the script does not work in php 8.2, an error message appears
HTTP ERROR 500
File rector.php:
_____________
`<?php

use Rector\Config\RectorConfig;
use Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector;

return RectorConfig::configure()
 
    // register single rule
    ->withRules([
        TypedPropertyFromStrictConstructorRector::class
    ])
    // here we can define, what prepared sets of rules will be applied
    ->withPhpSets(php82: true)
	->withoutParallel();`
_______

What could be the problem, where did I go wrong?

**Comment by samsonasik** [comment]
please stop spamming, you keep asking us to guess what's wrong on your project, while you not providing reproducible code, which nothing we can do.
